unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore.cs

```diff
@@ -20,7 +20,7 @@ public class DataStore
         public readonly DebugConfig debugConfig = new DebugConfig();
         public readonly BaseVariable<bool> isSignUpFlow = new BaseVariable<bool>();
         public readonly BaseDictionary<string, WearableItem> wearables = new BaseDictionary<string, WearableItem>();
-        public readonly BuilderInWorld builderInWorld = new BuilderInWorld();
+        public readonly DataStore_BuilderInWorld dataStoreBuilderInWorld = new DataStore_BuilderInWorld();
```

Comment:
Why have we changed the name from `builderInWorld` to `dataStoreBuilderInWorld`? Following the same rule as the rest of data stores we have, we should call it as `builderInWorld`.

Reply:
This was done automatically with the IDE when renaming the class, it was reverted in the next PR!